### PR TITLE
feat: add multi language support

### PR DESCRIPTION
**Pull request by chai2010**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [X] Y update #25  

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [X] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [X] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [X] Other, see https://crowdin.com/project/kusionstack

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/develop/build-docs/multi-lang.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# 文档多语言支持
+
+欢迎帮助将 KusionStack 文档翻译为其他语言。原始的文档是中文版本，目前最高优先级是翻译英文文档。
+
+多语言翻译工作通过 [crowdin](https://crowdin.com/project/kusionstack) 平台进行，最新的状态请参考 [Issue25](https://github.com/KusionStack/kusionstack.io/issues/25)。对应以下步骤：
+
+1. 本地修改了远文件：执行 `make i18n-upload` 上传源文件变更
+2. 登陆 crowdin 上的 KusionStack 翻译项目进行翻译：https://crowdin.com/project/kusionstack
+3. crowdin 上翻译文档文档更新：执行 `make i18n-download` 下载变更
+4. 执行  `make i18n` 查看效果：http://127.0.0.1:8080
```
[the remark this answers is not in the document]
Comment (chai2010) [reply docs/develop/build-docs/multi-lang.md]: fixed

**Review by Peefy — APPROVED** [review]
LGTM

**Review by Peefy — DISMISSED** [review]
LGTM

**Review by howieyuen — APPROVED** [review]
/lgtm

**Review by howieyuen — APPROVED** [review]

**Pull request merged**